.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+1. Click on '....'
+1. Scroll down to '....'
+1. See error
+
+**Actual results**
+Describe what happened
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Additional context**
```

Comment:
need to add what are the logs needs to be collected to report a bug

Reply:
i think we need below info also
what image version used 
what helm chat version used
kube version used
if requried kube templates

Reply:
Sure, adding those.